docs/source/introducing_v4.rst

```diff
@@ -114,18 +114,21 @@ you should use the ``get_raw_readout_data`` method to access the raw data and bu
 Using the new QPU Compiler Backend
 ----------------------------------
 
-Rigetti's next-generation QPU compiler is accessible through pyQuil v4. This new backend is still in development, so while it will eventually become the default, it is currently in limited access. If you have access, you can configure your compiler to use it using the new ``QPUCompilerAPIOptions`` class:
+Rigetti's next-generation QPU compiler is accessible through pyQuil v4. This backend is required for Ankaa based QPUs and can be configured with the new :py:class:`~pyquil.api.QPUCompilerAPIOptions` class.
 
 .. code:: python
 
     from pyquil.api import get_qc, QPUCompilerAPIOptions
     from pyquil.quil import Program
 
     program = Program()
-    qc = get_qc("Ankaa-1")
+    qc = get_qc("Ankaa-9Q-1")
 
     api_options = QPUCompilerAPIOptions()
-    api_options.use_backend_v2()
+    api_options.v2(
+        # See API docs for QPUCompilerAPIOptions for more information on this and other parameters
```

Comment:
Can we link?

Reply:
Two problems:

1. There isn't a great way to link inside of the code block as far as I can tell, but this isn't such a big deal because it is linked in the description above the code block.

2. Unfortunately, sphinx doesn't forward the docs from the aliased class _and_ it won't render the docstring for a type alias. I've updated the documentation here to link to the type hints for now, but once `qcs-sdk-python` has live docs, we can link to it directly.